packages/fx-core/src/component/generator/generatorAction.ts

```diff
@@ -153,12 +146,28 @@ export const unzipAction: GeneratorAction = {
   },
 };
 
+export const unzipAction: GeneratorAction = {
+  name: GeneratorActionName.Unzip,
+  run: async (context: GeneratorContext) => {
+    context.logProvider.debug(`Unzipping: ${JSON.stringify(context)}`);
+    if (context.zip) {
+      context.outputs = await unzip(
+        context.zip,
+        context.destination,
+        context.fileNameReplaceFn,
+        context.fileDataReplaceFn,
+        context.filterFn
+      );
+    }
+  },
+};
+
 export const TemplateActionSeq: GeneratorAction[] = [
   fetchTemplateZipFromSourceCodeAction,
   fetchTemplateUrlWithTagAction,
   fetchZipFromUrlAction,
-  fetchTemplateZipFromLocalAction,
   unzipAction,
+  fetchTemplateZipFromLocalAction,
```

Comment:
Let's update the action name. Maybe `fetchTemplateFromLocalAction`? You can think of a better name.

Reply:
Yeah, I think that name would be better.